### ISSUE TITLE
Number of rows

### ISSUE THREAD

**Issue by anu-a1** [abdennour/react-csv#32]
How many rows can this hold? 

I tried 2000 rows and it didn't work.  1000 rows works though.

When I try to download 2000, I get an empty file.

**Comment by diasdavid** [comment]
Same here. Anyway to increase the limit?

**Comment by isaklafleur** [comment]
I can do approx 11000 rows with 6 columns, but after that I get download error.

**Comment by benjaminbaldoni** [comment]
For those in need, or the lib author: 

I had the same problem when I was trying to export 20k+ rows, so I forked the repo and replaced the main function (which is the one that builds the file URL) to make use of blob (https://developer.mozilla.org/en-US/docs/Web/API/Blob), and that **fixed the problem** for me.

Since my understanding of blob is not complete I can't assure that it will work on every scenario, so I will not be creating a PR, but I'll leave the new code here - feel free to use it at will:

Original buildUri function (doesn't support large exports):
```
((data, headers, separator) => encodeURI(
  `data:text/csv;charset=utf-8,\uFEFF${toCSV(data, headers, separator)}`
)
```

New buildUri function (exporting working 👌)
```
(data, headers, separator) {
  const csvData = new Blob([toCSV(data, headers, separator)], { type: 'text/csv' }); 
    
  return URL.createObjectURL(csvData);
}
```

I found the solution on this SO question: https://stackoverflow.com/questions/23301467/javascript-exporting-large-text-csv-file-crashes-google-chrome

**Comment by phlare** [comment]
yeah i've got some issues with this as well, and would love for the above fix to be integrated into the real package.   larger data sets end up generating a link where the href value is SO large that it simply doesn't work

**Comment by cvburgess** [comment]
@benjaminbaldoni please do open a PR

**Comment by mccabemj** [comment]
The use of Blob was implemented in #73
